spaces/aliases.json

```
@@ -28,5 +28,6 @@
   "index": "index-coop.eth",
   "streamr": "streamr.eth",
   "rarible": "rarible.eth",
-  "curve": "curve.eth"
+  "curve": "curve.eth",
+  "chimpyswap": "chimpyswap.eth"
```

When creating a space you do not have to edit the `aliases.json` file.
```suggestion
  "curve": "curve.eth"
```